### ISSUE TITLE
Report issue with default styling of select in iOS Safari

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk-frontend#2526]
## What
As mentioned in [this issue comment](https://github.com/alphagov/govuk-frontend/issues/2435#issuecomment-1025544996) the default styling of select dropdowns in iOS Safari now displays as blue text on a grey background. Not only could this be interpreted as a disabled select dropdown, but it also fails colour contrast (see below).

<img width="728" alt="Screenshot 2022-01-31 at 13 57 38" src="https://user-images.githubusercontent.com/29889908/151985103-7363219f-5cb1-40ea-9640-077c4ab9128f.png">

![151769633-4b724119-d752-40c2-a064-f72032467ca4](https://user-images.githubusercontent.com/29889908/151986610-e38ad3ca-ceca-42f7-ac36-0f794f7e5f26.png)

## Why
While we've [implemented a fix to the Design System select component](https://github.com/alphagov/govuk-frontend/pull/2524) so that we have more control over the background and foreground colours of our select (as these issues seem to crop up quite frequently), we should also raise an issue with Webkit https://bugs.webkit.org/enter_bug.cgi

## Who needs to work on this
Developers / Accessibility Specialist

## Who needs to review this
Developers / Accessibility Specialist

## Done when

- [x] Bug report submitted on https://bugs.webkit.org/enter_bug.cgi , ideally with a screenshot and/or reproducible code snippet.
- [x] Issue created on https://github.com/alphagov/reported-bugs to track this

Examples of previous bug reports we've raised, for inspiration:

- https://bugs.webkit.org/show_bug.cgi?id=224746
- https://bugs.webkit.org/show_bug.cgi?id=224483
- https://bugs.webkit.org/show_bug.cgi?id=234935
